api/server.ts

```diff
@@ -23,18 +24,17 @@ import {
 } from './middleware/errorHandlers';
 /* eslint-enable import/first */
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-const handleError = (e: any): void => {
+const handleException = (e: Error | {} | null | undefined): void => {
   log.error(e);
   process.exit(1);
 };
 
-process.on('uncaughtException', handleError);
-process.on('unhandledRejection', handleError);
+process.on('uncaughtException', handleException);
+process.on('unhandledRejection', handleException);
 
 const app = express();
 
-if (ENV !== 'test') {
+if (!IS_TESTING) {
```

Comment:
@mycarrysun I changed this because it seemed like these lines would be interchangeable, but I just noticed that `IS_TESTING` is based on `NODE_ENV` instead of `ENV`. Will those variables be different (i.e, does this line need to change back?)

Reply:
Yea `NODE_ENV` is set at `npm test` see `package.json` for this piece `NODE_ENV = 'testing'`